File: wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/domain/wahlscheine/Wahlscheine.java
package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.wahlscheine;

import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkUndWahlID;
import jakarta.persistence.EmbeddedId;
import jakarta.persistence.Entity;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Wahlscheine {

    @EmbeddedId
    @NotNull
    private BezirkUndWahlID bezirkUndWahlID;

    @NotNull
    private long stimmabgabevermerke;
}

File: wls-gui-wahllokalsystem/src/types/ergebnismeldung/common/Wahlscheine.ts
import type { BezirkUndWahlID } from "@/types/ergebnismeldung/common/BezirkUndWahlID.ts";

export interface Wahlscheine {
  bezirkUndWahlID: BezirkUndWahlID;
  stimmabgabevermerke: number | null;
}
